### DISCUSSION TITLE
How to change unreal engine aircraft model？

### DISCUSSION THREAD

**Discussion by buseeryas** [JSBSim-Team/jsbsim#1012] [Q&A]
Jsbsimflightdynamicsmodel plug-in uses 787-8 model. How can change it to other types of aircraft models?I can't find the line of code to change the model..

**Comment by gallonmate** [comment]
It's defined in the blueprint. This image shows the details section once the blueprint is placed in the level editor and you can change the aircraft there. You can also edit the blueprint model name before placing it in the level editor.
![image](https://github.com/JSBSim-Team/jsbsim/assets/28702753/ccc2f9ba-817c-46d8-80e4-fe9aa5868b38)

**Reply by buseeryas** [reply]
Thank you!
